common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -1699,16 +1699,23 @@ def commits_details(base:, head:)
     end
   end
 
-  describe "#commit_message" do
+  describe "#commit_message", :vcr do
     subject(:commit_message) { builder.commit_message }
 
-    before do
-      allow(builder).to receive(:pr_name).and_return("PR name")
-      allow(builder).to receive(:commit_message_intro).and_return("Message")
-      allow(builder).to receive(:metadata_links).and_return("\n\nLinks")
```

Comment:
I should have noticed these stubs meant the test wasn't exercising the real diff generation code, removing them and using VCR to capture a fixture ensures we hit the code that was broken and test for values on the `_url` helpers that were moved out of this class.